google-cloud-build/ci-scripts/scripts/create_github_release.sh

```diff
@@ -9,12 +9,12 @@ then
   TAG_NAME="ci/${COMMIT:0:10}"
 fi
 BUILD_DIR=.build_output/cache/exports
-find "$BUILD_DIR" -type f -size +1900M | xargs rm
+find "$BUILD_DIR" -type f -size +1900M | xargs -0 rm
```

Comment:
Reason
^-- SC2038: Use -print0/-0 or -exec + to allow for non-alphanumeric filenames.

Reply:
The find needs -print0 as well, I think,

Reply:
Fixed